installers/olm/README.md

```diff
@@ -71,7 +71,7 @@ https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog
 
 
 Another version issue encountered was related to our attempt to both support OCP v4.6 (which is an Extended Update
-Support (EUS) release) while also limiting Kubernetes to 1.20+. The issue with this is that OCP 4.6 utilizes k8s 1.19
+Support (EUS) release) while also limiting Kubernetes to 1.21-1.24. The issue with this is that OCP 4.6 utilizes k8s 1.19
```

Comment:
⛏️ This should stay, I think.

Reply:
We encountered the issue when trying to limit it to `1.20+` but now we are limiting it to `1.21-1.24` :confused: 

I don't mind changing it back